signals.mss

```diff
@@ -682,19 +682,23 @@ Format details:
   /* DE block marker ("Blockkennzeichen) */
   /***************************************/
   ["feature"="DE-ESO:blockkennzeichen"] {
-    [zoom<16],
-    [zoom>=16][zoom<17][height > 1],
-    [zoom>=16][zoom<17][width > 2],
-    [zoom>=17][height > 2],
-    [zoom>=17][width > 4] {
       marker-file: url('symbols/de/blockkennzeichen.svg');
       marker-allow-overlap: true;
       marker-width: 14;
       marker-height: 14;
+
+    ::text {
```

Comment:
This block does not come into action because its label collides with the shield.

Reply:
Yeah, it's an ugly solution, but it looks fine on the map as far as I've seen.